frontend/src/lib/components/FlowBuilder.svelte

```diff
@@ -53,6 +55,7 @@
 	async function saveFlow(): Promise<void> {
```

Comment:
you're missing initialization at false when one is newly editing or adding a script (it will be in the components/route wrapping FlowBuilder)

Reply:
@fatonramadani that is still the case

Reply:
Done